scio-core/src/main/scala/com/spotify/scio/coders/KryoAtomicCoder.scala

```diff
@@ -123,6 +123,7 @@ final private class ScioKryoRegistrar extends IKryoRegistrar {
     k.forClass[LocalTime](new JodaLocalTimeSerializer)
     k.forClass[LocalDateTime](new JodaLocalDateTimeSerializer)
     k.forClass[DateTime](new JodaDateTimeSerializer)
+    k.forSubclass[Schema](new SchemaSerializer)
```

Comment:
To match the naming convention for other Kryo serializers, maybe let's call this `AvroSchemaSerializer`?